gestorpsi/client/tests.py

```diff
@@ -20,33 +20,52 @@
 
 from gestorpsi.client.models import Client, ClientManager
 from gestorpsi.client.models import Person
+from gestorpsi.service.models import Service, ServiceType, Area
 from gestorpsi.phone.models import Phone
-from gestorpsi.util.test_utils import user_stub, setup_required_data
+from gestorpsi.admission.models import ReferralChoice as AdmissionChoice
+from gestorpsi.organization.models import Organization
+from gestorpsi.authentication.views import login_check
+from gestorpsi.util.test_utils import client_stub, user_stub, setup_required_data, setup_required_service
 
 class ClientTests(TestCase):
     def setUp(self):
         self.factory = RequestFactory()
         # OBS: Using cl instead of client to evade conflicts with Client class
-        self.cl = Cl()
+        self.client = Cl()
```

Comment:
I recommend to use `cl` to refer to request client, and `client` to refer to gestorpsi client model (specifically on this package).